client_code/popover.py

```diff
@@ -0,0 +1,233 @@
+import anvil as _anvil
+from anvil.js.window import jQuery as _S, window as _window, document as _document
+"""
+    popover
+    Copyright 2020 Stu Cork
```

Comment:
Could we change the copyright statement to the anvilistas team members to match the rest of the repo?